### PR TITLE
Update copyright year in footer

### PR DESCRIPTION
**Pull request by Ein-Tim**
This tiny PR updates the copyright year in the footer to say "2020-2021" instead of "2020".

### CODE REVIEW

File: src/data/global.json
Comment (DerVogel2020) [inline src/data/global.json:301]: Here is a whitespace missing after 2021.

Comment (Ein-Tim) [reply src/data/global.json]: Thank you! Fixed in [`0174c6a`](https://github.com/corona-warn-app/cwa-website/pull/1102/commits/0174c6a236c9bfcdcb49f75aad68881fd836a07c).

**Comment by dsarkar** [comment]
@Ein-Tim Checking internally how to proceed with this. Thanks.

**Review by heinezen — APPROVED** [review]
LGTM and makes sense because the website updates a lot.

Waiting for internal approval

**Comment by dsarkar** [comment]
@Ein-Tim Approved! And thanks!

**Pull request merged**
